Src/Extern/SUNDIALS/AMReX_Sundials.H

```diff
@@ -1,8 +1,24 @@
 #ifndef AMREX_SUNDIALS_H_
 #define AMREX_SUNDIALS_H_
 
-#include <AMReX_SUNMemory.H>
+#include <AMReX.H>
+#include <AMReX_Arena.H>
 #include <AMReX_NVector_MultiFab.H>
 static_assert(std::is_same<amrex::Real,realtype>::value, "amrex::Real must be the same as SUNDIALS realtype");
+#include <AMReX_SUNMemory.H>
+
+namespace amrex {
+namespace sundials {
+
+void Initialize(int nthreads);
+void Finalize();
+#ifdef AMREX_USE_OMP
+::sundials::Context* The_Sundials_Context(int i = omp_get_thread_num());
+#else
+::sundials::Context* The_Sundials_Context(int i = 0);
+#endif
```

Comment:
Using a wrapper may be more concise:
https://github.com/AMReX-Codes/amrex/blob/development/Src/Base/AMReX_OpenMP.H
```suggestion
::sundials::Context* The_Sundials_Context(int i = amrex::OpenMP::get_thread_num());
```